exercises/space-age/canonical-data.json

```
@@ -80,6 +80,16 @@
         "seconds": 1821023456
       },
       "expected": 0.35
+    },
+    {
+      "uuid": "57b96e2a-1178-40b7-b34d-f3c9c34e4bf4",
+      "description": "age on Pluto",
```

```suggestion
      "description": "age on a non-planet",
```

Since the fact that it's Pluto specifically isn't important, and instead the important part is the fact that it's a non-planet, I suggest that this description better describes the true purpose of this test.

---

Why not use something nonsensical to test "unknown" or "not handled" such as "banana"?  No need to worry about it being a planet or a comet or anything that might orbit the sun?  The focus is not really "some nonexistent planet" but more "unknown input".